next.html

```diff
@@ -0,0 +1,625 @@
+                                                    data-asset="/content/dam/LeaderImpact/site-./assets/yt.svg"
+                                                    data-title="YouTube" id="image-46d1bf6845" class="cmp-image"
+                                                    itemscope="" itemtype="http://schema.org/ImageObject">
+                                                    <a href="#" class="cmp-image__link">
+                                                        <img src="./assets/yt.svg" class="cmp-image__image"
+                                                            itemprop="contentUrl" data-cmp-hook-image="image"
+                                                            alt="YouTube">
+                                                    </a>
+                                                </div>
+                                            </div>
+
+                                            <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <div data-cmp-lazythreshold="0"
+                                                    data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg"
+                                                    data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg"
+                                                    data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image"
+                                                    itemscope="" itemtype="http://schema.org/ImageObject">
+                                                    <a href="#" class="cmp-image__link">
+                                                        <img src="./assets/linkedin.svg" class="cmp-image__image"
+                                                            itemprop="contentUrl" data-cmp-hook-image="image"
+                                                            alt="LinkedIn">
+                                                    </a>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+          </div>
+          <!-- ABOUT BLURB -->
+          <div class="container responsivegrid blurb aem-GridColumn aem-GridColumn--default--12">
+            <div id="blurb-container" class="cmp-container" style="background-image:url(./assets/blurb-bg.png);">
```

Comment:
@Prystal254 We can't add inline styles. Please move this to the CSS file.

Reply:
The background image on the container component might do this inline actually

Reply:
@Prystal254 Bill is correct. We have the ability to add background colors and images with inline styles. When adding a background image, `background-size: cover; background-repeat: no-repeat` automatically gets added inline as well.